docs: repos README added (#6)
iiitv/resources: README.md

## [Design](design.md) 🎨

Apart from adding a visual flair to your software, apps or website, good design knowledge is required to also make accessible and easy to use apps. The easier it is to navigate across your product, the longer would the consumer want to stay.

## [Projects in IIITV](repos.md) 📘

> It contains all the open source projects currently active in this organisation. It holds numerous projects implemented in different stacks like React, Django among others.

Reviewer: ```suggestion
It contains all open source projects currently active in this organisation. It holds numerous projects implemented in different stacks like React, Django and many more.
```

Author: I suppose the `among others` is more appropriate

> It contains the all the open source projects currently active in this organisation. It holds numerous projects implemented in different stacks like React, Django among others.

Reviewer: ```suggestion
It contains all the open source projects currently active in this organisation. It holds numerous projects implemented in different stacks like React, Django among others.
```
